src/Reader/Http/ZendHttpClientDecorator.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Feed\Reader\Http;
+
+use Zend\Http\Client as ZendHttpClient;
+use Zend\Http\Headers;
+use Zend\Feed\Reader\Exception;
+
+class ZendHttpClientDecorator implements HeaderAwareClientInterface
```

Comment:
I don't agree with this assumption. 1 class - 1 test. This could has been highlighted the undefined variable used in the exception.